liblangutil/ErrorReporter.h

```diff
@@ -184,6 +190,8 @@ class ErrorReporter
 	bool checkForExcessiveErrors(Error::Type _type);
 
 	ErrorList& m_errorList;
+	
+	std::vector<std::string> m_errorMessages;
```

Comment:
Why do you need this? This does not seem to be used anywhere.

Also, modifying `ErrorReporter` does not seem like it should be needed to fix a bug like #13901.